packages/game/src/lib/who-played-card.test.ts

```diff
@@ -0,0 +1,42 @@
+import { describe, expect, it } from 'vitest';
+import { Board, Position } from './common-types';
+import { whoPlayedCard } from './who-played-card';
+
+describe('whoPlayedCard', () => {
+  const cases = [
+    [
+      'one',
+      [0, 0] as Position,
```

Comment:
Looks like we cast to `Position` often enough that we could write a `position` function that takes `x` and `y` and returns a `Position`. Called like `position(0, 0)` instead.